File: Python/birthday_cake_candles.py
'''
    birthdayCakeCandles has the following parameter(s):
        int candles[n]: the candle heights
    return int: the number of candles that are tallest
'''
def birthday_cake_candles(candles):
    candles.sort()
    tallest_candle = candles[len(candles)-1]
    return candles.count(tallest_candle)

#example
print(birthday_cake_candles([3,2,1,3]))

File: JS/birthdayCakeCandles.js
function birthdayCakeCandles(candles) {
    const orderedCandles = candles.sort((a,b) => {return a-b});
    let idCandle = candles.length-1

    const tallest = orderedCandles[idCandle];
    let unitsTallest = 0;

    while (orderedCandles[idCandle] == tallest){
        unitsTallest++
        idCandle--
    }
    
    return unitsTallest;
}

console.log(birthdayCakeCandles([3,2,1,3]))
